package.json

```
@@ -1630,93 +1630,105 @@
         "category": "Calva"
       },
       {
-        "command": "calva.refactor.cleanNs",
+        "command": "clojureLsp.refactor.cleanNs",
         "title": "Clean NS Form",
-        "category": "Calva Refactor",
+        "category": "clojure-lsp Refactor",
         "enablement": "editorLangId == clojure && clojureLsp:active"
       },
       {
-        "command": "calva.refactor.addMissingLibspec",
+        "command": "clojureLsp.refactor.addMissingLibspec",
         "title": "Add Missing Require",
-        "category": "Calva Refactor",
+        "category": "clojure-lsp Refactor",
         "enablement": "editorLangId == clojure && clojureLsp:active"
       },
       {
-        "command": "calva.refactor.cyclePrivacy",
+        "command": "clojureLsp.dragBackward",
```

I'd like it to be the same name as the calva counterpart, only with a different prefix/namespace.

```suggestion
        "command": "clojureLsp.dragSexprBackward",
```

---

Before I make this change... I want to double-check before we break a convention. The other command names are generated programmatically from the LSP command that is sent over the wire. That is `cycle-privacy`, which is sent to the LSP server, becomes  `cyclePrivacy` via
```javascript
command.replace(/-[a-z]/g, (m) =>
  m.substring(1).toUpperCase()          
```

Are you OK abandoning that convention for drag?

---

Hmmm, well, the convention is not all that important to me. I was thinking it would be easier to get the keybindings right if they commands had the same name and I just had to replace the prefix. (I got it wrong several times myself, but maybe I was tired and extra stupid.)

I do see how it makes the code a bit more predictable and easier... So let's skip harmonizing these names with their Calva equivalents. People should read the docs instead of guessing, anyway. 😄

---

I apologize, but I'm still not clear on what you want. `clojureLsp.dragSexprBackward` or `clojureLsp.dragBackward`?

---

I guess my words leak that I am very ambivalent about it. But let's go for `clojureLsp.dragBackward`

---

Gotcha... in that case, it's OK as is. I'm going to mark this conversation as resolved.